cobra/io/sbml.py

```diff
@@ -108,7 +108,7 @@ def create_cobra_model_from_sbml_file(sbml_filename, old_sbml=False,
             "convert fbc to cobra", True, "Convert FBC model to Cobra model")
         result = model_doc.convert(conversion_properties)
         if result != LIBSBML_OPERATION_SUCCESS:
-            raise(Exception("Conversion of SBML+fbc to COBRA failed"))
+            raise (Exception("Conversion of SBML+fbc to COBRA failed"))
```

Comment:
What do the brackets around the Exception do? Doen't `raise Exception(...)` work?